src/APlayer.js

```diff
@@ -154,70 +154,100 @@ class APlayer {
+                                    <div class="aplayer-volume-bar">
+                                        <div class="aplayer-volume" style="height: 80%;"></div>
+                                    </div>
                                 </div>
                             </div>
+                            <button type="button" class="aplayer-icon aplayer-icon-mode">
+                            </button>
+                            <button type="button" class="aplayer-icon aplayer-icon-menu">
+                            </button>
                         </div>
-                        <button type="button" class="aplayer-icon aplayer-icon-mode">
-                            ${this.getSVG(this.mode)}
-                        </button>
-                        <button type="button" class="aplayer-icon aplayer-icon-menu">
-                            ${this.getSVG('menu')}
-                        </button>
                     </div>
                 </div>
-            </div>
-            <div class="aplayer-list" ${this.option.listmaxheight ? `style="max-height: ${this.option.listmaxheight}` : ``}">
-                <ol>`;
-            for (let i = 0; i < this.option.music.length; i++) {
-                eleHTML += `
-                    <li>
-                        <span class="aplayer-list-cur" style="background: ${this.option.theme};"></span>
-                        <span class="aplayer-list-index">${(i + 1)}</span>
-                        <span class="aplayer-list-title">${this.option.music[i].title}</span>
-                        <span class="aplayer-list-author">${this.option.music[i].author}</span>
-                    </li>`
-            }
+                <div class="aplayer-list">
+                    <ol>`;
+            this.element.innerHTML = eleHTML;
+        }
+
+        eleHTML = `<ol>`;
+        for (let i = 0; i < this.option.music.length; i++) {
             eleHTML += `
-                </ol>
-            </div>`
-        this.element.innerHTML = eleHTML;
+                <li>
+                    <span class="aplayer-list-cur" style="background: ${this.option.theme};"></span>
+                    <span class="aplayer-list-index">${(i + 1)}</span>
+                    <span class="aplayer-list-title">${this.option.music[i].title}</span>
+                    <span class="aplayer-list-author">${this.option.music[i].author}</span>
+                </li>`
+        }
+        eleHTML += `</ol>`
+
+        this.element.querySelector('.aplayer-list').innerHTML += eleHTML;
+
+        if (this.music.pic) {
+            this.element.querySelector('.aplayer-pic').style.backgroundImage = `url('${this.music.pic}')`;
+        }
+
+        this.element.querySelector('.aplayer-icon-play').innerHTML += this.getSVG('play');
+        
+        // If not using Bootstrap style, use default background
+        // TODO: find a better way than hardcoding a very specific check
+        if (!this.element.querySelector('.progress')) {
```

Comment:
Thanks for the feedback. I'll close the pull request and just put these changes in my own fork.